### ISSUE TITLE
The server encountered a serious error

### ISSUE THREAD

**Issue by 01alex** [nlplab/brat#1352]
How to reproduce the behavior:

1. `git clone https://github.com/nlplab/brat.git`
2. `cd brat`
3. `./install.sh`
4. `python standalone.py`

The server starts at localhost:8001, shows the tutorial window, while giving the messages "The server encountered a serious error..." and "Unknown error: serverCrash" and I can't do anything else besides clicking "Ok" in the tutorial window.

The log in the terminal:

Serving brat at http://0.0.0.0:8001
Traceback (most recent call last):
  File "server/src/server.py", line 338, in serve
    return _safe_serve(params, client_ip, client_hostname, cookie_data)
  File "server/src/server.py", line 175, in _safe_serve
    from dispatch import dispatch
  File "server/src/dispatch.py", line 31, in <module>
    from norm import norm_get_data, norm_get_name, norm_search
  File "server/src/norm.py", line 12, in <module>
    from simstringdb import Simstring
  File "server/src/simstringdb.py", line 19, in <module>
    import simstringexec
  File "server/src/simstringexec.py", line 13, in <module>
    simstring_found = isfile(SIMSTRING_EXECUTABLE)
  File "/home/alexandre/Desktop/brat/venv/lib/python3.6/genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

Also, it throws the following error in the browser console:

Uncaught TypeError: Cannot read property 'length' of null
    at Function.each (jquery.min.js:2)
    at addAttributeTypesToDiv (annotator_ui.js:2134)
    at Function.rememberSpanSettings (annotator_ui.js:2236)
    at dispatcher.js:74

It works fine if I clone the repo at least at the "Fix for Python 3.8: cgi.escape moved to html.escape" commit.

**Comment by jeffmax** [comment]
I would try to install the simstring library or the simstring command line utility. I did the latter with 
`apt-get install simstring-bin`. There are some instructions here: https://github.com/nlplab/brat/blob/master/server/src/simstringdb.py#L1 but I was able to get it working without downloading that particular simstring version.

It looks like the previous version of brat used a python implementation of the simstring algorithm (simstring-pure), but it was slow, so it was recently removed in favor of using either a c library with python bindings or calling out directly to the simstring command line utility.
